lib/report_portal/tasks.rb

```diff
@@ -4,21 +4,22 @@
 require_relative '../reportportal'
 
 namespace :reportportal do
-  desc 'Start launch in Report Portal and print its id to $stdout (needed for use with ReportPortal::Cucumber::AttachToLaunchFormatter)'
+  desc 'Start launch in Report Portal and print its id to $stdout (for use with attach_to_launch formatter mode)'
   task :start_launch do
-    description = ENV['description']
-    file_to_write_launch_id = ENV.fetch('file_for_launch_id') { Pathname(Dir.tmpdir) + 'rp_launch_id.tmp' }
+    description = ReportPortal::Settings.instance.description || ENV['description']
```

Comment:
is the order correct? 
I think it should be
```suggestion
    description = ENV['description'] || ReportPortal::Settings.instance.description
```

Reply:
I think right given that `ReportPortal::Settings.instance.description` gets value from config YAML so config YAML would be prioritized over env variable without the prefix.